php_apcu/tests/test_php_apcu.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from datadog_checks.base import ConfigurationError
+from datadog_checks.php_apcu import PhpApcuCheck
+
+
+@pytest.mark.unit
+def test_config():
+    instance = {}
+    c = PhpApcuCheck('php_apcu', {}, [instance])
+
+    with pytest.raises(ConfigurationError):
+        c.check(instance)
+
+    c.check({'url': 'http://foobar'})
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_service_check(aggregator, instance):
+    c = PhpApcuCheck('php_apcu', {}, [instance])
+
+    c.check(instance)
+    aggregator.assert_service_check('php_apcu.can_connect', PhpApcuCheck.OK)
+
+    instance['url'] = instance['url'].replace('.php', '')
+    c.check(instance)
+    aggregator.assert_service_check('php_apcu.can_connect', PhpApcuCheck.CRITICAL)
```

Comment:
Let's put this in another test

Reply:
I looked at other tests and implemented it.
but APCu is after start-up it is almost no data.
sample output https://gist.github.com/withgod/58fca0210443b52cbec45e36a4c19ad1

Reply:
That's alright, I think having some assertion is good. But if the data will be inconsistent, we can consider adding a sample test fixture to mock the process of metric. We do something similar in our core integrations, like so:

Here we save the output to `test/fixtures/` directory: https://github.com/DataDog/integrations-core/blob/master/cilium/tests/fixtures/agent_metrics.txt

We create a pytest fixture that mocks the response: https://github.com/DataDog/integrations-core/blob/master/cilium/tests/conftest.py#L79-L90

And in a test, we pass the fixture in the parameters and it'll mock the `request.get` call with the response and assert metric collection: https://github.com/DataDog/integrations-core/blob/master/cilium/tests/test_cilium.py#L9-L15

Reply:
I think it is a good mechanism to be able to use fixture.
However, in this case, I don't think we need to use fixture because we can get it from the URL via docker.

If you're using more complex values, you should use fixture to pass complex values, but in this case, it's just a simple way to send what you get.

I thought you might be interested in the fact that I compared it to 0, so I annotated it.
I'll refer to this when I collect more complex metrics in the future. 😄